src/shaders/cube.js

```diff
@@ -0,0 +1,100 @@
+var registerShader = require('../core/shader').registerShader;
+var THREE = require('../lib/three');
+var utils = require('../utils/');
+
+var CubeLoader = new THREE.CubeTextureLoader();
+var texturePromises = {};
+
+/**
+ * Cubemap shader using THREE.ShaderMaterial.
+ */
+module.exports.Component = registerShader('cube', {
+  schema: {
+    src: { default: '' }
+  },
+  /**
+   * Initializes the shader.
+   * Adds a reference from the scene to this entity as the camera.
+   */
+  init: function (data) {
+    this.material = new THREE.ShaderMaterial(getMaterialData(data));
+    this.updateCubeMap(data);
+    return this.material;
```

Comment:
I just updated to not have to return this :) https://github.com/aframevr/aframe/pull/1549